### PR TITLE
Seller financing demo

### PR DESCRIPTION
**Pull request by vintlucky**
image

image

image

image

### CODE REVIEW

File: pages/seller_financing/index.tsx [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+                <tbody>
+                  {selectedDiscountedPayout && (
+                    <tr key={selectedDiscountedPayout.termKey}>
+                      <td>{toTermKeyLabel(selectedDiscountedPayout.termKey)}</td>
+                      <td>{formatCurrency(selectedDiscountedPayout.principal)}</td>
+                      <td>{formatCurrency(selectedDiscountedPayout.financingFee)}</td>
+                      <td>{selectedDiscountedPayout.discountFeePct.toFixed(2)}%</td>
+                    </tr>
+                  )}
+                  {/* {estimatedDiscountedPayout?.map((dp) => (
```
Comment (alvinsng) [inline pages/seller_financing/index.tsx:336]: delete this code?

File: components/MainHeader.tsx [hunk 2]
```diff
@@ -124,12 +132,12 @@ const MainHeader: React.FC<{
         }}
       >
         <Group>
-          <IconWallet />
+          {React.createElement(item.icon)}
```
Comment (alvinsng) [inline components/MainHeader.tsx]: You could do `<item.icon />` right?

File: pages/seller_financing/index.tsx [hunk 3]
```diff
@@ -0,0 +1,354 @@
...
+      fetchEstimatedDiscountedPayout(customer.id, customerSecret, total)
+    }
+  }, [customer?.id, total, financingTerm])
+
+  const initCustomer = async () => {
+    const customerResp = await fetch('/api/create-customer', {
+      method: 'POST',
+      body: JSON.stringify({
+        ...customerForm,
+        email: `test${CustomerType.SKIP_PRE_QUALIFY}@slopepay.com`,
```
Comment (alvinsng) [inline pages/seller_financing/index.tsx:100]: could we not use the email on the form?

File: components/MainHeader.tsx [hunk 4]
```diff
@@ -114,8 +117,8 @@ const MainHeader: React.FC<{
     ]
   }
 
-  const mItem = (
-    <Menu.Item>
+  const mItems = menuItems.map((item) => (
+    <Menu.Item key={item.link}>
```
Comment (alvinsng) [inline components/MainHeader.tsx]: const Icon = item.icon

**Review by alvinsng — APPROVED** [review]

File: pages/seller_financing/index.tsx (hunk 3 above)
Comment (vintlucky) [reply pages/seller_financing/index.tsx:100]: I don't want that complexity :\
the form's default is New customer

**Pull request merged**